### DISCUSSION TITLE
Do local models (LLM.CPP) have provider() classes in truera?

### DISCUSSION THREAD

**Discussion by YasaminAbbaszadegan** [truera/trulens#1000] [General]
Do local models (LLM.CPP) have provider() classes in truera?
from llama_index.llms.llama_cpp import LlamaCPP
llm = LlamaCPP(model_url='https://huggingface.co/TheBloke/Mistral-7B-Instruct-v0.2-GGUF/resolve/main/mistral-7b-instruct-v0.2.Q4_K_M.gguf', 
                   temperature=1, 
                   model_kwargs={"n_gpu_layers": -1},
                   verbose=True,)

--> from trulens_eval.feedback.provider import ?????

**Comment by joshreini1** [comment]
Local models can be run through the [LiteLLM](https://www.trulens.org/trulens_eval/api/provider/litellm/) or [Langchain](https://www.trulens.org/trulens_eval/api/provider/langchain/) feedback provider classes, or with a [custom feedback function](https://www.trulens.org/trulens_eval/evaluation/feedback_implementations/custom_feedback_functions/).

**Comment by joshreini1** [comment]
If you're willing to share an example with the community, please open up a PR. This would be a fantastic contribution!
